static/src/stylesheets/module/_social.scss

```diff
@@ -159,6 +165,13 @@ category: Common
     }
 }
 
+.social-icon--messenger {
+    background-color: $social-messenger;
+    &:hover {
+        background-color: darken($social-messenger, 10%);
```

Comment:
Thanks for `:focus` and `:active` states but I think, if we add them, we should add them for all social icons, and so in a separate pr.